CONTRIBUTING.md

```diff
@@ -194,3 +194,76 @@ IntelliJ may not pick up protoc generated classes as they can be very huge. If t
 
 ## CI
 The Ozone project uses Github Actions for its CI system.  The configuration is described in detail [here](.github/ci.md).
+
+## Debugging
+
+### Setup Ozone and Kubernetes
+
+Install a local server(k3s is recommended) and the Kubernetes command-line tool, kubectl. After setting up k3s and kubectl, build the project so that a SNAPSHOT image is created.
```

Comment:
This is linux specific, so lets just say, "You'll need kubernetes and kubectl installed."